wp-content/themes/missilethreat/assets/_scss/components/_key-takeaways.scss

```diff
@@ -0,0 +1,32 @@
+@use '../abstracts' as *;
+
+.key-takeaways {
+  margin-top: rem(32px);
+  margin-bottom: rem(56px);
+  padding: rem(24px) rem(36px) rem(32px) rem(36px);
+  background: $color-primary-light-gray;
+
+  &__title {
+    @extend %font-ui-heading-md;
+    margin-bottom: rem(12px);
+    color: $color-text-dark-100;
+  }
+  
+  &__list {
+    @extend %font-body-list-lg;
+    color: $color-text-dark-190;
+    list-style: initial !important;
```

Comment:
The [CSS Reset](https://github.com/hankchizljaw/modern-css-reset/issues/30) we use recently changed how they handle lists. It might be worth making this change in `_reset.scss` so you don't have to deal with this kind of specificity error later on, although be warned you should check to make sure that change doesn't impact anything else.